### DISCUSSION TITLE
Icon button + title and description

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1649] [Proposals]
### Goal

Necesitamos un componente icono en circulo + title y description para la nueva representación de entrypoint list.<img width="345" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/7f1eec0a-e703-4cc6-964e-051131beef41">

### Context

Para la new new app de Movistsar, debemos disponibilizar en el CMS un nuevo tipo de lista de entrypoint con aspecto de iconos circulares con titulo y descripción y que debe poder adaptarse a tener 1 o n elementos y funcionar bien sobre los posibles colores de fondo.
Actualmente se usa un componente que existe en desarrollo y que tiene un aspecto similar pero que es un bastardo creado de forma atómica. 

### Impact

Necesitamos este componente para la new new app de Movistar, lo que lo hace bastante urgente.

### Discussion Questions

¿Es posible que Mistica absorba esta necesidad o debemos optar por un extended?

### Do you have a visual proposal?

https://www.figma.com/file/7oGfCcnzS1okDr2xyRIE54/Entrypoints?type=design&node-id=4257-68235&mode=design&t=AelQUKVNsd8QcpyF-4

**Comment by aweell** [comment]
I close this issue since is already covered in #563.

Moving the comment to there.
